source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -9,43 +9,29 @@ parent: "/manual.html"
 - Sign in to the AWS console as an admin (`gds aws govuk-production-admin -l`, or however you prefer to sign in to AWS)
 - Sign in to [the GCP console](https://console.cloud.google.com/home/dashboard?project=govuk-production)
 - Open the following four pages as separate tabs:
   - [GCP Cloud DNS www-cdn.production.govuk.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/govuk-service-gov-uk/rrsets/www-cdn.production.govuk.service.gov.uk./CNAME/edit?project=govuk-production)
   - [AWS Route 53 govuk.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z22RPYZA77J620)
   - [GCP Cloud DNS assets.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/assets.publishing.service.gov.uk./CNAME/edit?project=govuk-production)
   - [AWS Route 53 publishing.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z3SBFBO09PD5HF)
-- Because the secondary CDN provides degraded service we have a grace period
-  before failing over
-  - Escalate to GOV.UK SMT as soon as you begin to consider failing over
-  - For a major Fastly outage (more than 30% of requests failing) wait 15
-    minutes (from the start of the incident) and then start the fail over
-  - For a minor Fastly outage (less than 30% of requests failing) wait at least
-    15 minutes (from the start of the incident) before failing over - discuss
-    the decision with the GOV.UK SMT Escalations person
-- While you wait, confirm the new CNAMES for www-cdn.production.govuk.service.gov.uk and assets.publishing.service.gov.uk
-  - This [Draft PR to Failover to AWS CloudFront](https://github.com/alphagov/govuk-dns-config/pull/714) shows the CNAMES you need to change, and how to test that they are correct
-  - You can also get the CNAMEs to use for the secondary CDN from the AWS CLI: <div class="highlight"><pre class="highlight plaintext"><code># www.gov.uk
-  gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='WWW'].DomainName | [0]"
-  # assets.publishing.service.gov.uk
-  gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='Assets'].DomainName | [0]"</code></pre></div>
-  - The records should look like `d0000000000000.cloudfront.net.` (with 0s replaced with letters and numbers)
 - When you are ready to fail over:
-  - Manually update the CNAME record for www-cdn.production.govuk.service.gov.uk in GCP and AWS
-  - Manually update the CNAME record for assets.publishing.service.gov.uk in GCP and AWS
+  - Manually update the CNAME record for `www-cdn.production.govuk.service.gov.uk` in both GCP and AWS to point to `d2uru0rbwu9sz0.cloudfront.net`
+  - Manually update the CNAME record for `assets.publishing.service.gov.uk` in both GCP and AWS to point to `d3d1psggl6xqso.cloudfront.net`
```

Comment:
Previously we were a bit coy about publishing the CloudFront URLs (even though they were effectively public, see `dig +short gov.uk TXT`), hence the complicated stuff with the AWS CLI. I think the worry was that they'd get used for web scraping or something, and we'd run up a big bill. Now that they're in front of origin instead of S3, they're potentially a new vector for DoS attacks, but making them secret doesn't exactly secure that threat.

I'm happy for them to be published if we think that we're managing the risk of them being misused in other ways (or that we've at least considered those threats).

Reply:
This is a good shout, I'll update this document to say "get the URLs from the linked PR" instead.